source/standards/accounts-with-third-parties.html.md.erb

```diff
@@ -0,0 +1,146 @@
+---
+title: Managing accounts with third party services
+expires: 2018-04-01
+---
+
+# <%= current_page.data.title %>
+
+<%= partial :expires %>
+
+When developing a service, your team will often require the use of
+third-party services.  To access these services, you will need some
+sort of user account.  This page provides guidance on how to manage
+those accounts.
+
+## Terms
+
+In this page we distinguish between an account and an individual:
+
+- An *account* means an account with an identifier such as a username or
+  email address
+- An *individual* means a human being
+- A set of *credentials* means some sort of secret used to gain access
+  to a user account (for example: a password, a secret access key)
+
+## Goals and principles
+
+When managing credentials we have a few goals:
+
+- Ensure that the right people have access
+- Ensure that the wrong people do not have access
+  - Particularly: ensure when people leave the organization, their
+    access is revoked or expired
+- Identify who did what and when (audit)
+  - This goal has varying importance depending on context
+  - Technical limitations may prevent the goal from being achieved, or
+    only provide partial audit
+
+To meet these goals, we have some principles:
+
+- Prefer to minimise the number of secrets that an individual needs to
+  keep
+  - This helps with revocation because there are fewer sets of
+    credentials to revoke when someone leaves
+  - Passwords are particularly problematic secrets. Other forms of
+    secrets (eg secret access keys or OAuth tokens) are more easily
+    revoked and recycled.
+- No individual should ever have to use credentials associated with
+  another individual
+  - This would be counter to the audit goal
+- There should be no need to share credentials between teams
+  - Again, this is problematic for audit
+  - This is also problematic for knowing when credentials should be
+    revoked
+  - Technical limitations may mean this is unavoidable sometimes
+
+## Creating accounts for shared third-party services
+
+The recommendations here depend on whether the service supports
+organisation or team access natively.
+
+### Services that support organisation accounts
+
+For services which support organisation accounts, we should use this
+to ensure the correct individuals have access.  Therefore:
+
+- Create separate accounts for each individual rather than sharing
+  credentials
+- Prefer using a single-sign-on provider over creating an account with
+  separate username and password
+- Prefer using G-Suite as a single sign-on provider over any other
+  provider
+    - this is because G-Suite is a core part of the leaver's process and
+      so we have high confidence this will be revoked when somebody
+      leaves
```

Comment:
I also heard the argument that GDS IT can take control of the user account if necessary e.g. the only person with access is on holiday when a crisis occurs